repositories.md

```diff
@@ -235,3 +235,14 @@ In setting up an Uptane system, an implementer will need to specify how informat
 
 Information on writing a POUF can be found on the [POUF Purpose and Guidelines](https://uptane.github.io/pouf.html) page of the Uptane website. A sample POUF, written for the [Uptane Reference Implementation](https://uptane.github.io/reference_pouf.html), offers sample metadata written in [ASN.1/DER](https://github.com/uptane/uptane.github.io/blob/master/reference_pouf.md#file-formats).
+
+## Cost Considerations
+
+There is no short answer to the question of what it costs to implement Uptane because such a decision has a number of variables to be considered, starting with how they plan to do this implementation.  Currently, an  OEM has two methods to implement Uptane: either buy an off-the-shelf solution and do an integration, or build a custom solution, with some greater or smaller degree of reliance on the open-source client and server components. If a project is starting from scratch, there is a third option. Uptane could be integrated in the design stage. At that point, the cost would be minimal and any money spent would be just one component of the initial design. But, most OEMS contemplating an integration will not be at that initial stage. 
+
+If a user is contemplating options one or two, the relative costs become admittedly more difficult to estimate. In the case of off-the-shelf products, the goals of the provider and the OEM could be somewhat misaligned, making the integration difficult and likely expensive. Additionally, there is always a risk profile associated with getting an OTA system from a 3rd party, as the supplier could go out of business, or decide to phase out the product in the wake of a shift in corporate priority.
+
+Arguably then, some variant of the custom in-house option is the safest choice for larger OEMs. But it will mean higher cost and greater effort. 
+
+Ultimately, the issue of cost cannot be decided without also considering the value received for the expense. Value in this case is enhanced security, so when discussing costs you must look at the trade-off between taking shortcuts and sacrificing security, or doing it right and spending more time/money. For example, robust key rotation is one of the most important pillars of TUF and Uptane, but it is also something that companies may be prepared to ignore if they think it will save money. Therefore, before thinking about cost, potential implementers should do a risk assessment of the current solution and evaluate the current weaknesses and identify the major opportunities for improvement.  Only by knowing the risks they are defending against can implementers legitimately assess if the cost is an investment in the security of their project and their company rather than just another expense eating away at profits. 
```

Comment:
```suggestion
Ultimately, the issue of cost cannot be decided without also considering the value received for the expense. Value in this case is enhanced security, so when discussing costs, the trade-off between taking shortcuts and sacrificing security, or doing it right and spending more time and money, must be considered. For example, robust key rotation is one of the most important pillars of TUF and Uptane, but it is also something that users may be prepared to ignore if they think it will save money. Therefore, before thinking about cost, potential implementers should do a risk assessment of the current solution and evaluate the current weaknesses and identify the major opportunities for improvement.  Only by knowing the risks they are defending against can implementers legitimately assess if the cost is an investment in the security of their project and their company rather than just another expense eating away at profits.
```